src/drivers/driver.ts

```diff
@@ -665,7 +665,24 @@ export abstract class CMakeDriver implements vscode.Disposable {
         CMakeExecutableVersion: cmakeVersion ? `${cmakeVersion.major}.${cmakeVersion.minor}.${cmakeVersion.patch}` : '',
         CMakeGenerator: this.generatorName || '',
         ConfigType: this.isMultiConf ? 'MultiConf' : this.currentBuildType || '',
+        Toolchain: this._kit?.toolchainFile ? "true" : "false", // UseToolchain?
+        Trigger: trigger
       };
+
+      if (this._kit?.compilers) {
+        if (this._kit.compilers["C"]) {
+          telemetryProperties.CCompiler = this._kit.compilers["C"];
```

Comment:
This needs to be scrubbed for PII. We can only report on known compilers and must exclude the path information.  cpptools has a list if you want to grab from that.  I can show you where it is.  I think we might want to simplify and remove version numbers from compiler names in favor of querying the version and logging that too.